ui/components/multichain/token-list-item/token-list-item.tsx

```diff
@@ -100,6 +102,15 @@ export const TokenListItem = ({
   const metaMetricsId = useSelector(getMetaMetricsId);
   const isMetaMetricsEnabled = useSelector(getParticipateInMetaMetrics);
   const isMarketingEnabled = useSelector(getDataCollectionForMarketing);
+  const { safeChains } = useSafeChains();
+
+  const decimalChainId = isEvm && parseInt(hexToDecimal(chainId), 10);
+
+  const [safeChainDetails] = safeChains
```

Comment:
Would `.find` work here instead?

Reply:
Could also be worth expanding the condition from `safeChains` to `safeChains && typeof decimalChainId === 'number'` to skip for non-EVM chains?

Reply:
https://github.com/MetaMask/metamask-extension/pull/26994/commits/74ebf605f41b8210cc20e15756a21dd5eb2daf21